use_cases/gretel.json

```diff
@@ -307,6 +307,46 @@
         "label": "Launch Gretel Labs",
         "link": "https://image-synthetics-preview.gretel.cloud/"
       }
+    },
+    {
+      "gtmId": "use-case-synthetic-database",
+      "title": "Synthesize a database",
+      "description": "Synthesize a database using Gretel Relational.",
+      "cardType": "Console",
+      "icon": "relational-db.png",
+      "modelType": "synthetics",
+      "modelCategory": "synthetics",
+      "defaultConfig": "config_templates/gretel/synthetics/tabular-actgan.yml",
+      "sampleConnection": {
+        "id": "sample_mysql_telecom",
+        "type": "mysql",
+        "description": "Use this sample telecommunications database to synthesize an entirely new database of statistically equivalent records, while maintaining referential integrity.",
+        "tables": 5,
+        "records": 230988,
+        "fields": 29,
+        "trainingTime": "20 mins",
+        "bytes": 9537000
+      }
+    },
+    {
+      "gtmId": "use-case-transform-database",
+      "title": "Redact PII in a database",
+      "description": "Safeguard sensitive information and uphold compliance standards by automatically removing private information from your database.",
+      "cardType": "Console",
+      "icon": "relational-db.png",
+      "modelType": "transform",
+      "modelCategory": "transform",
+      "defaultConfig": "config_templates/gretel/transform/default.yml",
```

Comment:
Let's make the default config for this transform v2 now that it is out of beta.

Reply:
you'll want to change both modelType and defaultConfig for that ;)